inc/container.class.php

```diff
@@ -487,7 +489,7 @@ public function getValueToSelect($field_id_or_search_options, $name = '', $value
                 return Dropdown::showFromArray($name, self::getTypes(), $options);
             case $this->getTable() . '.itemtypes':
                 $options['display'] = false;
-                return Dropdown::showFromArray($name, self::getItemtypes(), $options);
+                return Dropdown::showFromArray($name, self::getItemtypes(false), $options);
```

Comment:
I'm not really sure about this one; I do not know if there is a way here to find what to pass

Reply:
It seems OK.